_posts/2020-03-15-learning-to-share-accomplishments-1.md

```diff
@@ -0,0 +1,96 @@
+hunting. More specifically, I mention one of the pieces of feedback was on how I come off as a bit more of an 
+"individual contributor" (someone who just takes tasks and does them) as opposed to someone who actually designs or 
+engineers solutions (someone who takes a vague thing and maps how how it will work, breaks it apart into smaller 
+tasks, then can work on some of those). 
+
+After some thought, I realized this is because over time I've learned to give credit to those on my team for some of 
+the things we've worked on. I think it feels better to have a team take credit (well, and take on the failures) of all 
+aspects of a project. While this is important and I think every team should do this, it sucks for a technical job 
+interviews. It doesn't work to "sell yourself" to a future employer.
+
+For part 1 of this post, I want to spend some time thinking through how to talk about accomplishments. What's the right 
+way to do it without feeling like I have an inflated ego? How do I own up to my work, even when I've been on teams?
+Why does it feel so hard to do?
+
+## "Bragging" About Yourself
+
+As someone who now has been in dozens of conferences, podcasts, videos, websites, blogs/articles, etc., I've had to 
+come to terms with tooting my own horn some. All of these things usually want a bio from me. It's a lot easier now that 
+I have made multiples and keep them around for things like this. In the beginning, it wasn't easy. In fact it was just 
+downright terrible. It just felt wrong almost to try to write about all these things. It felt like bragging.
+
+The easiest way I found to toot my own horn? I found one rule makes it easy: **Stick to the facts.**
+
+The basics are to look at things I actually accomplished factually. These things definitively are provable. They have 
+a place, a time, a number, or just anything that is solid and concrete about them. These aren't disputable because 
+they're not my opinions. I _did_ work that job. I _was_ on that team. I _wrote_ that code for _that_ project. Because 
+I can't dispute these, it makes it easier to say about myself.
+
+But the basics of a bio are easy. Actually taking a team accomplishment and diving into my specific thing is a bit 
+harder (at least mentally).
+
+For each project/team/accomplishment, what was my one specific contribution? In some cases this is easy, like when I 
+implemented a feature for a project that made it into production. Other times less so, like when I built a thing and 
+learned it later got scrapped. The scrapping wasn't my fault, and it feels weird to talk about it because it isn't 
+visible anymore. It's still an accomplishment of mine though, and it's one of the things I'm trying to keep an open 
+mind about when writing this post.
+
+Finally, what specifics can I back up things with? Did something I build save the team time? Did it save the company 
```

Comment:
This is something I struggled with for a long time, even coming from a background where it was *all* numbers. 
I think this level of detail is perfect for this post, but I think there would be value in another post about how to quantify your contributions. 👍

Reply:
Or perhaps an additional blurb to do it?